src/robotide/ui/tagdialogs.py

```diff
@@ -304,15 +312,14 @@ def add_tag(self, tag_to_tests):
 
         This method associates the internal model data with the GUI. While the
         sort order of the GUI can change at any time, the order of the data in
-        the model does not.
-        The entry in the model is referenced by a new id generated by
-        wx.NewId(). This prevents overwriting entries when combining delete and
-        insert statements.
+        the model does not. The entry in the model is referenced by a new id
+        generated by wx.NewId(). This prevents overwriting entries when
+        combining delete and insert statements.
 
         Args:
             index: An int marking the position to insert the row at.
             tag_to_tests: A tuple mapping tests(list, index 0) to a
-            tag(str, index 1).
+                          tag(str, index 1).
```

Comment:
Oh, I see. It was indented to match `: ` on the above line. I guess it makes sense. Having changes like this in same commits as actual features is nevertheless confusing.